microsoft/botframework-solutions
# Enterprise bot localization

## Description

These changes support localization of responses in the Enterprise Bot Template:
- Added SetLocaleMiddleware
- Updated MainResponses.cs to pull localized Intro card and suggested actions
- Fixed unsupported syntax in Intro.json cards

## Related Issue




resolves #423 

## Testing Steps


- Pull down branch
- Copy contents of ...\AI\templates\Enterprise-Template\src\csharp\EnterpriseBotTemplate to your ...\Documents\Visual Studio 2017\Templates\ProjectTemplates folder
- Open VS, create a new project using "Enterprise Bot Template"
- Ensure project builds
- Run msbot clone deployment (or add existing enterprise bot .bot file to project and update appsettings.json)
- Verify you can see localized intro cards when changing the locale setting in emulator

## Checklist
~~- [ ] I have added or updated the appropriate unit tests~~
~~- [ ] I have updated related documentation~~

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~~- [ ] A duplicate issue is filed to track future  work.~~

If you have updated responses or `.lu` files:
~~- [ ] You have tested deployment with your new models~~
